src/images/icons/cup.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
I removed `fill` from here (comparing to mobile icon), should I actually do that @coderitual?

Reply:
great, we don't use `fill` one color icons